config.yml

```
@@ -72,18 +78,19 @@ steps:
         with: 03_move-done-local.md
         issue: Uploading your project
         data:
-          issueURL: '%actions.new_issue.data.html_url%'
+          issueURL: '{{ store.fourth_issue_url }}'
   #4
   - title: Public or private repository?
     description: Understand the difference between a public and private repository.
     event: issues.closed
-    link: '{{ repoUrl }}/issues'
+    link: '{{ store.fourth_issue_url }}'
     actions:
       - type: createIssue
-        action_id: new_issue
         title: A few parting words
         body: 04_graceful-exit.md
+        store:
+          fifth_issue_url: {{ results.data.html_url }}
```

I think this needs to be:

```suggestion
          fifth_issue_url: '{{ results.data.html_url }}'
```

Unfortunate though 🤔

---

@JasonEtco thank you! Thankfully, the bot caught it too and I fixed it in the latest commit :wink: